app/assets/javascripts/models/beta/note.js

```diff
@@ -2,14 +2,18 @@ import httpService from '../../services/httpService';
 import changeCase from 'change-object-case';
 import { setLoadingValue } from './story';
 
-export const destroy = (projectId, storyId, noteId) =>
-  httpService
+export const destroy = async (projectId, storyId, noteId) =>
+  await httpService
```

Comment:
Do you really need this `async` and `await` here? 

```js 
export const destroy = (projectId, storyId, noteId) => 
  httpService.delete(`/projects/${projectId}/stories/${storyId}/notes/${noteId}`)
```

I believe this code ☝️ will have the same behavior since `httpService.delete` will returns a promise. 🤔

Reply:
:o